flutter/example/lib/main.dart

```diff
@@ -120,6 +120,17 @@ class MainScaffold extends StatelessWidget {
                 Sentry.captureMessage('A message with a print() Breadcrumb');
               },
             ),
+            RaisedButton(
```

Comment:
Will this be deleted before finalizing it? I assume the idea is to run this on all outgoing events, right?

Reply:
Yes, it's currently more in a proof of concept stage. I'm also still searching for available information. So not plugging this into a proper place in the SDK makes things a little easier.

There are also this Accessibility information available:
https://api.flutter.dev/flutter/dart-ui/AccessibilityFeatures-class.html 

Yeah, I'll attach a screenshot 👍